Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm

```diff
@@ -96,7 +97,10 @@
 
 bool RemoteScrollbarsController::shouldRegisterScrollbars() const
```

Comment:
We should invent a name for "having scrollbars that need to render in the web process" and use it here and instead of exposing `isInsideSVGForeignObject` in more places.

Reply:
Changed this to having scrollable area have a shouldRenderScollbarsInProcess.